src/main/python/flaskApp/api/extraction/controllers.py

```diff
@@ -32,3 +34,46 @@ def extractText():
         "Status": "Citizen"
     }
     return jsonify({"Extracted Fields": text})
+
+
+@extract.route('/extractFace', methods=['POST'])
+def extractFace():
+    """
+    ----------------------------------------------------------------------
+    Author: Nicolai van Niekerk
+    ----------------------------------------------------------------------
+    Sample function to extract face from image received
+    ----------------------------------------------------------------------
+    URL: http://localhost:5000/extractFace
+    ----------------------------------------------------------------------
+    """
+    image = request.files.get("image")
```

Comment:
This is fine for mock purposes but we have to make sure that the images are passed in a format that we can convert to numpy arrays at the moment this is not possible.